ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -159,6 +159,197 @@ objects:
+                  path: /etc/rollback-test
+        EOF
+          oc create -f /tmp/machineconfig.yaml
+
+          function wait_for_machineconfigpool_to_apply() {
+            for i in $(seq 0 10); do oc wait machineconfigpool/master --for=condition=Updating --timeout=5m && break; done
+            for i in $(seq 0 10); do oc wait machineconfigpool/master --for=condition=Updated --timeout=5m && break; sleep 30; done
+          }
+
+          wait_for_machineconfigpool_to_apply
+
+          echo "Setting up ssh bastion"
+          mkdir -p ~/.ssh || true
+          cp "${KUBE_SSH_KEY_PATH}" ~/.ssh/id_rsa
+          chmod 0600 ~/.ssh/id_rsa
+          echo "${USER:-default}:x:$(id -u):$(id -g):Default User:$HOME:/sbin/nologin" >> /etc/passwd
+          curl https://raw.githubusercontent.com/eparis/ssh-bastion/master/deploy/deploy.sh | bash
+          BASTION_HOST=$(oc get service -n openshift-ssh-bastion ssh-bastion -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+
+          function bastion_ssh() {
+            while true
+            do
+              ssh -o LogLevel=error -o ConnectionAttempts=100 -o ConnectTimeout=5 -o StrictHostKeyChecking=no -o ProxyCommand="ssh -A -o StrictHostKeyChecking=no -o LogLevel=error -o ServerAliveInterval=30 -o ConnectionAttempts=100 -o ConnectTimeout=5 -W %h:%p core@${BASTION_HOST}" $@ && break
+              sleep 10
+            done
+          }
+
+          echo "Make etcd backup on first master"
+          FIRST_MASTER=$(oc get node -l node-role.kubernetes.io/master= -o name | head -n1 | cut -d '/' -f 2)
+          cat > /tmp/etcd_backup.sh <<'EOF'
+            #!/bin/bash
+            set -x
+            RUNNING_ETCD_POD=$(crictl pods -q --label k8s-app=etcd --state=Ready)
+            RUNNING_ETCD_CONTAINER=$(crictl ps --pod ${RUNNING_ETCD_POD} --name etcd-member -q)
+            crictl exec ${RUNNING_ETCD_CONTAINER} /bin/sh -c 'source /run/etcd/environment && ETCDCTL_API=3 etcdctl --cert /etc/ssl/etcd/system:etcd-peer:${ETCD_DNS_NAME}.crt --key /etc/ssl/etcd/system:etcd-peer:${ETCD_DNS_NAME}.key --cacert /etc/ssl/etcd/ca.crt snapshot save /var/lib/etcd/snapshot.db'
+        EOF
+          chmod +x /tmp/etcd_backup.sh
+          scp -o StrictHostKeyChecking=no -o ProxyCommand="ssh -A -o StrictHostKeyChecking=no -o ServerAliveInterval=30 -W %h:%p core@${BASTION_HOST}" /tmp/etcd_backup.sh "core@${FIRST_MASTER}":/tmp
+          bastion_ssh "core@${FIRST_MASTER}" "sudo -i /tmp/etcd_backup.sh"
+
+          echo "Backup manifests"
+          mapfile -t MASTERS < <(oc get node -l node-role.kubernetes.io/master= -o name | cut -d '/' -f 2)
+          for master in "${MASTERS[@]}"
+          do
+            bastion_ssh "core@${master}" "sudo -i mkdir /etc/kubernetes/manifests-backup && sudo -i cp -rvf /etc/kubernetes/manifests/* /etc/kubernetes/manifests-backup"
+          done
+
+          # echo "Upgrade to latest CI release"
+          # oc adm upgrade --to-image "${RELEASE_IMAGE_LATEST}" --force
```

Comment:
I think you can drop this block now, and we can bring it back if/when we decide to add "restore pre-upgrade snapshots after an upgrade" testing.

Reply:
Right, this would need to be reworked as `setup` container would need to be reconfigured